src/main/webapp/WEB-INF/tags/instructor/search/searchPageInput.tag

```diff
@@ -32,22 +32,22 @@
                 class="glyphicon glyphicon-info-sign">
             </span>
           </li>
-          <li>
-            <input id="comments-for-responses-check" type="checkbox"
-                name="<%=Const.ParamsNames.SEARCH_COMMENTS_FOR_RESPONSES%>" value="true"
-                <c:if test="${data.searchCommentForResponses}">checked</c:if>>
-            <label for="comments-for-responses-check">
-              Comments for responses
-            </label>
-          </li>
           <li>
             <input id="students-check" type="checkbox"
-                name="<%=Const.ParamsNames.SEARCH_STUDENTS%>" value="true"
-                <c:if test="${data.searchForStudents || !data.searchCommentForResponses}">checked</c:if>>
+                   name="<%=Const.ParamsNames.SEARCH_STUDENTS%>" value="true"
+                   <c:if test="${data.searchForStudents || !data.searchCommentForResponses}">checked</c:if>>
```

Comment:
Fix these indentation errors

Reply:
Done, I had `Align attributes` checked in IntelliJ and it affected the indentation